apps/api/src/app/widgets/dtos/message-response.dto.ts

```diff
@@ -201,6 +201,9 @@ export class MessagesResponseDto {
   @ApiProperty()
   totalCount: number;
 
+  @ApiProperty()
+  hasMore: boolean;
```

Comment:
missing tests for this flag

Reply:
Good catch, thanks! 🤘 
I create a test for the pagination itself but not the flag, I will add one as well.